### ISSUE TITLE
Figure 5 - style transfer diagram - overlapping text but only sometimes

### ISSUE THREAD

**Issue by redblobgames** [distillpub/post--differentiable-parameterizations#92]
[Figure 5](https://distill.pub/2018/differentiable-parameterizations/#figure-style-transfer-diagram) looks fine in Chrome 70 (MacOS) and Safari 12 (MacOS) and Safari 12 (iOS) when the browser width is 1180px or greater but 1179px and below the words on the overlap:

<img width="201" alt="Figure 5 overlapping text" src="https://user-images.githubusercontent.com/8757369/47958733-65bc7600-df8e-11e8-8d62-98049da9c96e.png">

On my Firefox 64 (Mac OS) the text overlaps even at the wider browser width, but my Firefox is highly customized so I can't be sure it's not something else.

On Chrome 70 (Windows 10) “content objective” and “aims” run together at all widths for me, but there's no change at 1180px.

IE/Edge (Windows 10) is the only browser I tried where it always looks right (!).

**Comment by redblobgames** [comment]
I'm not sure where the text is being built but if you put all the `tspan` elements on a line without `x` and `y` on each one after the first, it seems to put it in the right place:

    <tspan x="629" y="12.3">The </tspan>
    <tspan font-weight="500">content objective</tspan>
    <tspan> aims </tspan>

**Comment by ludwigschubert** [comment]
Good catch, I thought I had found all of these—the underlying issue here was SF Pro Display vs SF Pro Text… never trust software to do your typesetting.
